docs/DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -127,6 +134,8 @@ Optional, but strongly recommended:
 
 7. For eclipse formatted plugin, there is an issue for Idea: <https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter> - `cannot save settings Path to custom eclipse folder is not valid` - it works only when settings were saved from down to up. And not vice versa.
 
+If something is still not working, try to clean up (delete all from domains/sormas/autodeploy, domains/sormas/applications, domains/sormas/generated, and domains/sormas/osgi-cache) try to build again by executing `mvn clean install -DskipTests` on the `sormas-base` module
```

Comment:
Stop the payara domain before cleaning, run Ant `deploy-serverlibs` to update libs before starting the domain and deploying again